recipe/meta.yaml

```diff
@@ -15,24 +15,24 @@ source:
   sha256: 6d60e03e1abd30a7d4afe547d157ce3dd7a8c166fc5e407fd6d62ae99ff30460
 
 build:
-  number: 0
+  number: 1
```

Comment:
You need `merge_build_host: True   # [win]` here

Reply:
Thanks! I'll try to submit that to the `conda smithy update-cb3`.